synspec_examples.ipynb

```diff
@@ -0,0 +1,722 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from synphot.spectrum import SourceSpectrum\n",
+    "from synphot.models import Empirical1D\n",
+    "\n",
+    "photlam_hatp11 = SourceSpectrum(Empirical1D,\n",
+    "                                points=wavelengths, lookup_table=flux)\n",
+    "\n",
+    "# Scale the flux to get the value at Earth\n",
+    "photlam_hatp11 = photlam_hatp11 * float(stellar_radius / distance) ** 2 / np.pi\n",
+    "\n",
+    "photlam_hatp11.plot(flux_unit='PHOTLAM', left=0, right=60000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(for other methods of constructing source spectra with `synphot`, see the first bulleted list <a href=\"https://synphot.readthedocs.io/en/latest/synphot/spectrum.html#arrays\">here</a>) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"spectroscopy\"></a>\n",
+    "## 4. Simulate an observation with a spectrometer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will model the spectrometer after the instrument on the planned <a href=\"https://en.wikipedia.org/wiki/CHEOPS\">CHEOPS satellite</a>, setting a spectral resolution R = 1000."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 154,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "aperture_radius = 32 / 2 * u.cm  # radius of 32 cm CHEOPS satellite\n",
+    "aperture_area = np.pi * aperture_radius ** 2\n",
+    "R = 1000  # spectral resolution lambda"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The CHEOPS spectrometer will observe in wavelengths ranging from 0.33 - 1.1 microns:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 155,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# set a constant delta_wavelength defined by R and the wavelength in the center of the range\n",
+    "wl_in_center = ((1.1 - 0.33) / 2 + 0.33)\n",
```

Comment:
Why not apply unit here? You apply the unit in the next cell anyway.
```suggestion
    "wl_in_center = ((1.1 - 0.33) / 2 + 0.33) * u.micron\n",
```

Reply:
In reply to your response above, yeah, I think `np.arange` doesn't play well with Quantity, so if you go this route, you would have to use `delta_wavelength.value` to calculate `spec_wavelengths`. Neither way is more beautiful than the other. 🤔

Reply:
See https://github.com/tcjansen/GSoC/pull/17#issuecomment-511563088